src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -87,63 +87,63 @@
+    protected static final int RESOURCES_NAME_EXTRA_CHARS = 10;
 
-    private static final String VIRTUAL_NETWORK_NAME_BASE = "vnet";
+    protected static final String VIRTUAL_NETWORK_NAME_BASE = "vnet";
 
-    private static final String PUBLIC_IP_ADDRESS_NAME_BASE = "ip";
+    protected static final String PUBLIC_IP_ADDRESS_NAME_BASE = "ip";
 
-    private static final String NETWORK_SECURITY_GROUP_NAME_BASE = "sg";
+    protected static final String NETWORK_SECURITY_GROUP_NAME_BASE = "sg";
 
-    private static final String NETWORK_INTERFACE_NAME_BASE = "if";
+    protected static final String NETWORK_INTERFACE_NAME_BASE = "if";
 
-    private static final String OS_DISK_NAME_BASE = "os";
+    protected static final String OS_DISK_NAME_BASE = "os";
 
-    private static final Boolean DEFAULT_STATIC_PUBLIC_IP = true;
+    protected static final Boolean DEFAULT_STATIC_PUBLIC_IP = true;
 
-    private static final String SCRIPT_EXTENSION_PUBLISHER = "Microsoft.Azure.Extensions";
+    protected static final String SCRIPT_EXTENSION_PUBLISHER = "Microsoft.Azure.Extensions";
 
-    private static final String SCRIPT_EXTENSION_TYPE = "CustomScript";
+    protected static final String SCRIPT_EXTENSION_TYPE = "CustomScript";
 
-    private static final String SCRIPT_EXTENSION_VERSION = "2.0";
+    protected static final String SCRIPT_EXTENSION_VERSION = "2.0";
 
-    private static final String SCRIPT_EXTENSION_CMD_KEY = "commandToExecute";
+    protected static final String SCRIPT_EXTENSION_CMD_KEY = "commandToExecute";
 
-    private static final String SCRIPT_SEPARATOR = ";";
+    protected static final String SCRIPT_SEPARATOR = ";";
 
-    private static final String SINGLE_INSTANCE_NUMBER = "1";
+    protected static final String SINGLE_INSTANCE_NUMBER = "1";
 
-    private static final String INSTANCE_NOT_FOUND_ERROR = "ERROR unable to find instance with ID: ";
+    protected static final String INSTANCE_NOT_FOUND_ERROR = "ERROR unable to find instance with ID: ";
 
     @Autowired
-    private AzureServiceCache azureServiceCache;
+    protected AzureServiceCache azureServiceCache;
 
     @Autowired
-    private AzureProviderUtils azureProviderUtils;
+    protected AzureProviderUtils azureProviderUtils;
 
     @Autowired
-    private AzureProviderNetworkingUtils azureProviderNetworkingUtils;
+    protected AzureProviderNetworkingUtils azureProviderNetworkingUtils;
 
     @Autowired
-    private TagManager tagManager;
+    protected TagManager tagManager;
 
     @Value("${connector-iaas.azure.default-username:activeeon}")
-    private String defaultUsername;
+    protected String defaultUsername;
 
     @Value("${connector-iaas.azure.default-password:Act1v€0N}")
-    private String defaultPassword;
+    protected String defaultPassword;
 
-    @Value("${connector-iaas.azure.default-private-network-cidr:10.0.0.0/24}")
-    private String defaultPrivateNetworkCidr;
+    @Value("${connector-iaas.azure.default-protected-network-cidr:10.0.0.0/24}")
```

Comment:
The change in the property name above is a mistake in the auto-refactoring. You need to revert it to connector-iaas.azure.default-private-network-cidr so that:
(1) it has the same meaning as the field
(2) more importantly, it has the same name as in the `application.properties` configuration file.

Reply:
Thanks, done.